CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.39.3
```

Comment:
Should be `0.40.0` since we are adding a feature. I also usually add `(provisional)` at the right so that people know until the version is actually released.

Reply:
✅